<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SceneView</class>
 <widget class="QWidget" name="SceneView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEditSearch">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="placeholderText">
        <string>Filter</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonAdd">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>45</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="../gameplay-editor.qrc">
         <normaloff>:/res/images/scene-add.png</normaloff>:/res/images/scene-add.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="SceneTreeView" name="treeView">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::InternalMove</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::MoveAction</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="autoExpandDelay">
      <number>-5</number>
     </property>
     <property name="indentation">
      <number>20</number>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="animated">
      <bool>true</bool>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionAdd_Node">
   <property name="icon">
    <iconset resource="../gameplay-editor.qrc">
     <normaloff>:/res/images/scene-node.png</normaloff>:/res/images/scene-node.png</iconset>
   </property>
   <property name="text">
    <string>Add Node</string>
   </property>
   <property name="toolTip">
    <string>Add Node</string>
   </property>
  </action>
  <action name="actionAdd_Cube">
   <property name="icon">
    <iconset resource="../gameplay-editor.qrc">
     <normaloff>:/res/images/scene-shape-cube.png</normaloff>:/res/images/scene-shape-cube.png</iconset>
   </property>
   <property name="text">
    <string>Add Cube</string>
   </property>
   <property name="toolTip">
    <string>Add Cube</string>
   </property>
  </action>
  <action name="actionAdd_Cone">
   <property name="icon">
    <iconset resource="../gameplay-editor.qrc">
     <normaloff>:/res/images/scene-shape-cone.png</normaloff>:/res/images/scene-shape-cone.png</iconset>
   </property>
   <property name="text">
    <string>Add Cone</string>
   </property>
   <property name="toolTip">
    <string>Add Cone</string>
   </property>
  </action>
  <action name="actionAdd_Cylinder">
   <property name="icon">
    <iconset resource="../gameplay-editor.qrc">
     <normaloff>:/res/images/scene-shape-cylinder.png</normaloff>:/res/images/scene-shape-cylinder.png</iconset>
   </property>
   <property name="text">
    <string>Add Cylinder</string>
   </property>
   <property name="toolTip">
    <string>Add Cylinder</string>
   </property>
  </action>
  <action name="actionAdd_Sphere">
   <property name="icon">
    <iconset resource="../gameplay-editor.qrc">
     <normaloff>:/res/images/scene-shape-sphere.png</normaloff>:/res/images/scene-shape-sphere.png</iconset>
   </property>
   <property name="text">
    <string>Add Sphere</string>
   </property>
   <property name="toolTip">
    <string>Add Sphere</string>
   </property>
  </action>
  <action name="actionAdd_Camera">
   <property name="icon">
    <iconset resource="../gameplay-editor.qrc">
     <normaloff>:/res/images/scene-camera.png</normaloff>:/res/images/scene-camera.png</iconset>
   </property>
   <property name="text">
    <string>Add Camera</string>
   </property>
   <property name="toolTip">
    <string>Add Camera</string>
   </property>
  </action>
  <action name="actionAdd_SpotLight">
   <property name="icon">
    <iconset resource="../gameplay-editor.qrc">
     <normaloff>:/res/images/scene-light-spot.png</normaloff>:/res/images/scene-light-spot.png</iconset>
   </property>
   <property name="text">
    <string>Add Spot Light</string>
   </property>
   <property name="toolTip">
    <string>Add Spot Light</string>
   </property>
  </action>
  <action name="actionAdd_PointLight">
   <property name="icon">
    <iconset resource="../gameplay-editor.qrc">
     <normaloff>:/res/images/scene-light-point.png</normaloff>:/res/images/scene-light-point.png</iconset>
   </property>
   <property name="text">
    <string>Add Point Light</string>
   </property>
   <property name="toolTip">
    <string>Add Point Light</string>
   </property>
  </action>
  <action name="actionAdd_DirectionalLight">
   <property name="icon">
    <iconset resource="../gameplay-editor.qrc">
     <normaloff>:/res/images/scene-light-directional.png</normaloff>:/res/images/scene-light-directional.png</iconset>
   </property>
   <property name="text">
    <string>Add Directional Light</string>
   </property>
   <property name="toolTip">
    <string>Add Directional Light</string>
   </property>
  </action>
  <action name="actionAdd_Terrain">
   <property name="icon">
    <iconset resource="../gameplay-editor.qrc">
     <normaloff>:/res/images/scene-terrain.png</normaloff>:/res/images/scene-terrain.png</iconset>
   </property>
   <property name="text">
    <string>Add Terrain</string>
   </property>
   <property name="toolTip">
    <string>Add Terrain</string>
   </property>
  </action>
  <action name="actionAdd_Sprite">
   <property name="icon">
    <iconset resource="../gameplay-editor.qrc">
     <normaloff>:/res/images/scene-sprite.png</normaloff>:/res/images/scene-sprite.png</iconset>
   </property>
   <property name="text">
    <string>Add Sprite</string>
   </property>
   <property name="toolTip">
    <string>Add Sprite</string>
   </property>
  </action>
  <action name="actionAdd_Text">
   <property name="icon">
    <iconset resource="../gameplay-editor.qrc">
     <normaloff>:/res/images/scene-text.png</normaloff>:/res/images/scene-text.png</iconset>
   </property>
   <property name="text">
    <string>Add Text</string>
   </property>
   <property name="toolTip">
    <string>Add Text</string>
   </property>
  </action>
  <action name="actionAdd_Form">
   <property name="icon">
    <iconset resource="../gameplay-editor.qrc">
     <normaloff>:/res/images/scene-form.png</normaloff>:/res/images/scene-form.png</iconset>
   </property>
   <property name="text">
    <string>Add Form</string>
   </property>
   <property name="toolTip">
    <string>Add Form</string>
   </property>
  </action>
  <action name="actionAdd_ParticleEmitter">
   <property name="icon">
    <iconset resource="../gameplay-editor.qrc">
     <normaloff>:/res/images/scene-particles.png</normaloff>:/res/images/scene-particles.png</iconset>
   </property>
   <property name="text">
    <string>Add Particle Emitter</string>
   </property>
   <property name="toolTip">
    <string>Add Particle Emitter</string>
   </property>
  </action>
  <action name="actionAdd_Audio">
   <property name="icon">
    <iconset resource="../gameplay-editor.qrc">
     <normaloff>:/res/images/scene-audio.png</normaloff>:/res/images/scene-audio.png</iconset>
   </property>
   <property name="text">
    <string>Add Audio</string>
   </property>
   <property name="toolTip">
    <string>Add Audio</string>
   </property>
  </action>
  <action name="actionAdd_Video">
   <property name="icon">
    <iconset resource="../gameplay-editor.qrc">
     <normaloff>:/res/images/scene-video.png</normaloff>:/res/images/scene-video.png</iconset>
   </property>
   <property name="text">
    <string>Add Video</string>
   </property>
   <property name="toolTip">
    <string>Add Video</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="../gameplay-editor.qrc">
     <normaloff>:/res/images/scene-add.png</normaloff>:/res/images/scene-add.png</iconset>
   </property>
   <property name="text">
    <string>Add to Scene</string>
   </property>
   <property name="shortcut">
    <string>Meta+N</string>
   </property>
  </action>
  <action name="actionDelete_Node">
   <property name="icon">
    <iconset resource="../gameplay-editor.qrc">
     <normaloff>:/res/images/delete.png</normaloff>:/res/images/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete Node</string>
   </property>
   <property name="shortcut">
    <string>Shift+Del</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SceneTreeView</class>
   <extends>QTreeView</extends>
   <header>SceneTreeView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../gameplay-editor.qrc"/>
 </resources>
 <connections/>
</ui>
